src/BTVNanoCommissioning/workflows/ctag_eWc_valid_sf.py

```diff
@@ -143,37 +144,25 @@ def process_shift(self, events, shift_name):
         ## Soft Muon cuts
         soft_muon = events.Muon[softmu_mask(events, self._campaign)]
         req_softmu = ak.count(soft_muon.pt, axis=1) >= 1
+        mujetsel = (ak.all(event_jet.metric_table(soft_muon) <= 0.4, axis=2)) & (
+            (event_jet.muonIdx1 != -1) | (event_jet.muonIdx2 != -1)
+        )
+        mujetsel2 = (
+            ak.all(
+                events.Jet.metric_table(soft_muon) <= 0.4,
+                axis=2,
+            )
+        ) & ((events.Jet.muonIdx1 != -1) | (events.Jet.muonIdx2 != -1))
```

Comment:
The `mask_identity=True` should be there to avoid `None`  events pass `True` criteria. 
 The `ak.fill_none(mujetsel,False,axis=-1)` then remove `None`, this cut can be proceed anywhere. (Then you don't need `mujetsel2`)